mw-webapp/src/component/accordion/Accordion.cy.tsx

```diff
@@ -0,0 +1,94 @@
+import {Accordion, accordionTypes} from "src/component/accordion/Accordion";
+import {AccordionContent} from "src/component/accordion/AccordionContent/AccordionContent";
+import {AccordionTrigger} from "src/component/accordion/AccordionTrigger/AccordionTrigger";
+
+describe("Accordion component", () => {
+  const ACCORDION_ITEMS_EXAMPLE = [
+    {
+      trigger: <AccordionTrigger
+        text="Trigger1"
+        dataCy="trigger1"
+      />,
+      content: <AccordionContent
+        content={<div>
+          Content1
+        </div>}
+        dataCy="content1"
+      />,
+    },
+    {
+      trigger: <AccordionTrigger
+        text="Trigger2"
+        dataCy="trigger2"
+      />,
+      content: <AccordionContent
+        content={<div>
+          Content2
+        </div>}
+        dataCy="content2"
+      />,
+    },
+  ];
+
+  it("should render the accordion and all options", () => {
+    cy.mount(<Accordion items={ACCORDION_ITEMS_EXAMPLE} />);
+
+    cy.get("[data-cy=content1]").should("exist");
+    cy.get("[data-cy=content2]").should("exist");
+    cy.get("[data-cy=trigger1]").should("exist");
+    cy.get("[data-cy=trigger2]").should("exist");
+  });
+
+  it("should accordion option be closed (text hidden)", () => {
+    cy.mount(<Accordion items={ACCORDION_ITEMS_EXAMPLE} />);
+
+    cy.get("[data-cy=content1]").should("be.not.visible");
+  });
+
+  it("should accordion option be opened when click trigger", () => {
+    cy.mount(<Accordion items={ACCORDION_ITEMS_EXAMPLE} />);
+    cy.get("[data-cy=trigger1]").click();
+
+    cy.get("[data-cy=content1]").should("be.visible");
+  });
+
+  it("should all options could be opened and closed one by one", () => {
+    cy.mount(<Accordion items={ACCORDION_ITEMS_EXAMPLE} />);
+
+    cy.get("[data-cy=trigger1]").click();
```

Comment:
Let's create a little helper to simplify access to elements via data-cy attributes, it could be useful when you will have different data attributes for some purposes in future
 ```const dataCy = (value:string) => `[data-cy=£{value}]`)```

Reply:
Let's use constants instead of strings. It is very hard to maintain strings

Reply:
Done.